core/arch/instrlist.c

```diff
@@ -245,7 +245,8 @@ check_translation(instrlist_t *ilist, instr_t *inst)
 #if defined(CLIENT_INTERFACE) && defined(ARM)
     if (instr_is_meta(inst)) {
         dr_pred_type_t auto_pred = ilist->auto_pred;
-        if (auto_pred != DR_PRED_AL && auto_pred != DR_PRED_NONE) {
+        if (auto_pred != DR_PRED_AL && auto_pred != DR_PRED_OP &&
```

Comment:
I added `DR_PRED_OP` here because drmemtrace required it, though I'm not quite sure what it is. I do not see this predicate labeled [here](http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.den0024a/CHDEEABE.html).

Should we keep this `DR_PRED_OP` check here?